uint8 axis
uint8 requestedState
bool isCalibration
---
bool success